### PR TITLE
analyze: do not attempt to set up cgroups while verifying units

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

**Comment by keszybz** [comment]
bionic-s390x: boot-and-services  `FAIL stderr: Failed to dump process list for 'systemd-nspawn@c1.service', ignoring: Unit systemd-nspawn@c1.service not loaded.`.  Unrelated.

centosci (arch linux + sanitizers): `test-execute     FAIL`.  I'm not quite sure, but I think if this was caused by this patch, there would be more failures.

**Comment by keszybz** [comment]
@mrc0mmand could  you take a look?

File: src/core/manager.c [hunk 1]
```diff
@@ -870,7 +870,7 @@ int manager_new(UnitFileScope scope, ManagerTestRunFlags test_run_flags, Manager
         if (r < 0)
                 return r;
 
-        if (test_run_flags == MANAGER_TEST_RUN_MINIMAL) {
+        if (FLAGS_SET(test_run_flags, MANAGER_TEST_RUN_MINIMAL)) {
```
Comment (poettering) [inline src/core/manager.c:873]: there are some other places where we check for the flag with ==. Shouldn't they be updated too?

Comment (bluca) [reply src/core/manager.c:873]: Yes there's two more, but they seemed more intentional, judging by which tests are setting the flag to be exactly == MANAGER_TEST_RUN_MINIMAL, so I opted to leave them as-is as I was not 100% sure

**Comment by mrc0mmand** [comment]
> @mrc0mmand could you take a look?

This is definitely unrelated. There's something which causes the test-execute to timeout under sanitizers, but so far I haven't found what. I'll temporarily skip the test under sanitizers, since it's been affecting quite a few builds.

**Pull request merged**